### PR TITLE
Dce fixes eloy/v1

### PR DESCRIPTION
**Pull request by inashivb**
#578 
#571 
#569 
#568

### CODE REVIEW

File: tests/dcerpc/dcerpc-dce-iface-many/test.rules [hunk 1]
```diff
@@ -0,0 +1 @@
+alert dcerpc any any -> any any (msg: "DCE Netlogon"; dce_iface: 12345678-1234-abcd-ef00-01234567cffb; sid: 1;)
```
Comment (pevma) [inline tests/dcerpc/dcerpc-dce-iface-many/test.rules:1]: Not sure if it matters for the test but is it worth doing it with `dcerpc.iface` and `dcerpc.opnum` keywords (respectively below) instead as those are the new/sticky buffer stanza ?

Comment (inashivb) [reply tests/dcerpc/dcerpc-dce-iface-many/test.rules:1]: indeed. Thanks, Peter!

**Comment by victorjulien** [comment]
This fails on all branches. Should it?

**Comment by zer1t0** [comment]
Yes, it should only work after apply the fixes in https://github.com/OISF/suricata/pull/6725

**Comment by inashivb** [comment]
> This fails on all branches. Should it?

I'll check the behavior of these parsers pre-rust, 6.0.x should require a fix.

**Comment by zer1t0** [comment]
Yes, the branch 6.0.x required also the fixes. Here they were:
https://github.com/OISF/suricata/pull/6499
https://github.com/OISF/suricata/pull/6497

**Comment by inashivb** [comment]
> Yes, the branch 6.0.x required also the fixes. The following PR introduces the fixes (new ones will be required without the C unittests): [OISF/suricata#6499](https://github.com/OISF/suricata/pull/6499) [OISF/suricata#6497](https://github.com/OISF/suricata/pull/6497)

Thank you very much, @zer1t0 ! I had overlooked these PRs, will use these to prepare a PR for 6.0.x.

**Comment by inashivb** [comment]
Replaced w #620 

When https://github.com/OISF/suricata/pull/6725 gets merged, it will be backported to the `6.0.x` branch and these tests be updated.

**Pull request closed without merging**